### ISSUE TITLE
Problems with dockerized Rstudio working directory to run TM functions

### ISSUE THREAD

**Issue by YonitMMS** [mpicbg-scicomp/tissue_miner#48]
Hello,

I have encountered a few different problems with using TM through Rstudio.
When I use the dockerized version (opened as instructed in https://mpicbg-scicomp.github.io/tissue_miner/user_manual/TM_R-UserManual.html#prepare-the-movie-data), I have access to all TM functions, but don't seem to be able to access the folders where my data is held (I think this is because the working directory can't change from home to another directory).  I thought maybe copying my data to this home directory will solve the problem, but don't know how to access it. 
From my local (undockerized) RStudio, I can of course change the working directory and access my data, but is it possible to set up RStudio to use TM this way also in Windows? This option doesn't seem to appear in the manual for Windows. 
I will emphasize that I have been running the quickstart tutorial functions through docker on my data no problem, but would like to use the broader options available through RStudio.

Another issue I have encountered when trying to install and use TM on a different computer. It seems like the download of the TM docker image works ok, but when trying to run the "tm sm make_db" command, permission is denied. Does the data need to be saved in a specific folder?

I understand these issues may be not directly related to TM, and may be quite basic, but nevertheless would like to raise them here.

Thanks,
Yonit Maroudas-Sacks

**Comment by etournay** [comment]
Hi Yonit, 
Could you give me the specs of the Windows (7 or 10, home/family/professionnel/server) and the Docker version installed? Could you also tell me which shell you use (powershell, cygwing, etc...)?
There is indeed very limited section dedicated to  Windows because we don't have windows machines. In principle it works but the settings may be tricky.  With this info I try to get back to you this week-end.

**Comment by YonitMMS** [comment]
Hi, both computers I am working on have Windows 10 Enterprise 2015, and Docker Quickstart Terminal (which has since been deprecated, but it seems we cannot download the newer Docker toolbox on these computers).
It seems that the second issue I wrote about is solved by saving the files in certain folders instead of others, so please don't worry about that part.  So, to make it clear, I'm able to run tm commands through the docker terminal, but not through RStudio.
Thanks very much!

**Comment by etournay** [comment]
Not easy, but I suspect that if you use the old Docker Quickstart Terminal you have Virtualbox installed. If it is the case, then you need to specifically include the shares (D: or other mount points)  directly in Virtualbox: Settings > Shared folders
Once this is done in Virtualbox, Docker will be able to see the shares too. I hope this helps.
Another possibility is to install a Linux in a Virtualbox and run docker from the virtualized Linux. I did that a lot for testing purposes. If your hardware is good enough that should be an acceptable solution regarding performances...

**Comment by YonitMMS** [comment]
Hi Raphael, the first option does indeed work, so now everything seems to be working smoothly. Thanks very much for your help!
Yonit
